src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/JudgmentValidationServiceTest.java

```diff
@@ -37,6 +41,18 @@ void populateJudgmentDateOfHearingTest() throws ParseException {
         caseDetails1.getCaseData().getJudgementCollection().get(0).getValue().getDynamicJudgementHearing().setValue(caseDetails1.getCaseData().getJudgementCollection().get(0).getValue().getDynamicJudgementHearing().getListItems().get(0));
         judgmentValidationService.validateJudgments(caseDetails1.getCaseData());
         Assertions.assertEquals("2019-11-01", caseDetails1.getCaseData().getJudgementCollection().get(0).getValue().getJudgmentHearingDate());
+    }
 
+    @Test
+    void populateJudgmentZeroHearings() throws ParseException {
+        var caseData = caseDetails1.getCaseData();
+        caseData.setHearingCollection(null);
+        DynamicJudgements.dynamicJudgements(caseData);
+        var dynamicValue = DynamicListHelper.getDynamicValue(NO_HEARINGS);
+        Assertions.assertEquals(dynamicValue, caseData.getJudgementCollection().get(0).getValue().getDynamicJudgementHearing().getListItems().get(0));
```

Comment:
I'd just import the static Assertions.xxx rather than always write Assertions. everywhere.

Reply:
refactored